src/rustllvm/PassWrapper.cpp

```diff
@@ -188,7 +188,7 @@ LLVMRustCreateTargetMachine(const char *triple,
     }
 
     TargetOptions Options;
-    Options.PositionIndependentExecutable = PositionIndependentExecutable;
+    //Options.PositionIndependentExecutable = PositionIndependentExecutable;
```

Comment:
Did you mean we have a `LLVMRustSetModulePIELevel` with the `#ifdef` for calling the `setPIELevel` method?
If so, at what point do we call it?